src/PhpWord/Style/Cell.php

```diff
@@ -356,4 +376,92 @@ public function getNoWrap(): bool
     {
         return $this->noWrap;
     }
+
+    /**
+     * Get style padding-top.
+     *
+     * @return mixed
+     */
+    public function getPaddingTop()
+    {
+        return $this->paddingTop;
+    }
+
+    /**
+     * Set style padding-top.
+     *
+     * @return $this
+     */
+    public function setPaddingTop(int $value): self
```

Comment:
@Azamat8405 Could you add unit tests for setter and getter in [tests/PhpWordTests/Style/CellTest.php](https://github.com/PHPOffice/PHPWord/blob/master/tests/PhpWordTests/Style/CellTest.php) ?